pkg/storage/segments/repairer.go

```diff
@@ -85,7 +85,7 @@ func (repairer *Repairer) Repair(ctx context.Context, path storj.Path) (err erro
 	// repair not needed
 	if int32(numHealthy) > pointer.Remote.Redundancy.RepairThreshold {
 		mon.Meter("repair_unnecessary").Mark(1)
-		return Error.New("segment %v with %d pieces above repair threshold %d", path, numHealthy, pointer.Remote.Redundancy.RepairThreshold)
+		return nil
```

Comment:
Lets add a log line about that, but then LGTM

Reply:
unfortunately there is no logger available :/

Reply:
it would be easy to make one accessible as part of this PR
https://github.com/storj/storj/pull/2365/files

Reply:
Can you take care of that once this is merged? @mobyvb

Reply:
Yeah, I'll add it to the PR. Should only be a few lines